scripts/listResourceTypes.ts

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env -S tsx
+import {
+  CloudFormationClient,
+  ListStacksCommand,
+  DescribeStackResourcesCommand,
+  StackSummary,
+} from "@aws-sdk/client-cloudformation";
+import { STSClient, GetCallerIdentityCommand } from "@aws-sdk/client-sts";
+import { mkdirSync, writeFileSync } from "fs";
+import { join } from "path";
+
+const region = "us-east-1";
+const cf = new CloudFormationClient({ region });
+const sts = new STSClient({ region });
+
+const NAME_FILTERS = ["main", "master", "val", "prod"];
```

Comment:
SEDS prod is "production" I believe

Reply:
For full completion, all of them are production except QMR which is "prod". I hope to have that sorted in the next few weeks so we can be 3 environments 3 names across all apps.

Reply:
I don't think that's a real risk. I am just hopeful that we could do exact matching on this sort of thing in the future.

Reply:
This has been fixed.